extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ConnectionModel.java

```diff
@@ -165,6 +172,28 @@ public Integer getMaxConnectionsPerUser() {
     }
 
     /**
+     * Sets the connection weight.
+     *
+     * @param connectionWeight
+     *     The weight of the connection.  null is acceptable, and anything
```

Comment:
Hopefully the updated commentary here looks better.  I didn't know exactly how to describe this since it should be coming from the DB - let me know if you're looking for something more specific.